.github/workflows/docs.yml

```diff
@@ -24,14 +24,14 @@ jobs:
         with:
           version: 'v0.8.0'
 
-      - name: Compile sa.typ
-        run: typst compile SA/sa.typ
+      - name: Compile documents
+        run: make all
 
-      - name: Upload sa.pdf
+      - name: Upload design_concept.pdf
```

Comment:
update name of the task to somthing like `Upload artifacts` (this time `name` is for display...)